src/app/mock/mock-http.ts

```diff
@@ -247,10 +247,6 @@ export class MockHttp extends HttpService {
           } else {
             return this.createResponse(url.toString(), 200, 'ok', { data: this.mockDataService.getAllAreas() });
           }
-        case '/source-link-types':
```

Comment:
I've added @sanbornsen I've added the mock endpoint in 	8eba828 and also removed a link category that was included in the link type's `included` array but doesn't belong to the link type in 9f9aff8.